pkg/api/v1/types.go

```diff
@@ -202,6 +202,8 @@ type VolumeSource struct {
 	PersistentVolumeClaim *PersistentVolumeClaimVolumeSource `json:"persistentVolumeClaim,omitempty" description:"a reference to a PersistentVolumeClaim in the same namespace; see http://releases.k8s.io/HEAD/docs/user-guide/persistent-volumes.md#persistentvolumeclaims"`
 	// RBD represents a Rados Block Device mount on the host that shares a pod's lifetime
 	RBD *RBDVolumeSource `json:"rbd,omitempty" description:"rados block volume that will be mounted on the host machine; see http://releases.k8s.io/HEAD/examples/rbd/README.md"`
+	// DownwardAPI represents an host volume which contains pod's info supplied by downward API
+	DownwardAPI *DownwardAPIVolumeSource `json:"downwardApi,omitempty" description:"Downward API volume contains information about the pod"`
```

Comment:
Please read:
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#naming-conventions

It was recently updated. Acronyms should be of all the same case. The JSON field name should therefore be downwardAPI.